### DISCUSSION TITLE
4.3.26 statue spawns not working, single player works fine

### DISCUSSION THREAD

**Discussion by anthraxmonki** [Pryaxis/TShock#1722] [General]
Hello,

Not a single statue will spawn their respective entities using the tshock server world.

I took an SCP of the tshock world, ran single player local on that world, and the statue spawns work perfectly fine.

tshock configs are vanilla.

Did I miss something? Or is there a bug

**Comment by hakusaro** [comment]
It could be a bug? Can you post any more information about it (like the precise version you used and tested against)?
